Source/Fuse.Common/Timer.uno

```diff
@@ -29,8 +29,8 @@ namespace Fuse
 
 		void Stop()
 		{
-			_running = false;
 			UpdateManager.RemoveAction(Update);
+			_running = false;
```

Comment:
If you mean that there is no point in setting `_running` to false, i'd argue there is at it makes `_running` accurately represent the current state of the Timer. This might avoid unexpected behavior for the next person making changes relying on `_running` to Timer